### ISSUE TITLE
Browser slowing down with new develop branch

### ISSUE THREAD

**Issue by shpoudel** [GRIDAPPSD/gridappsd-forum#67]
@poorva1209 @bibiraju 

I am having an issue of **slowing down of the browser** and see that the simulation status in the viz seems to be lagged significantly. With this, the application performance and the corresponding simulation in the platform are not synchronized well.  I have attached an image in this post as well.

P.S: I have checked the real-time button in the simulation configuration.

Please let me know if you can think of any possible solution.

![image](https://user-images.githubusercontent.com/35777492/79606872-1ba65200-80a7-11ea-8f86-84786521e639.png)

**Comment by nhuyvan** [comment]
@bibiraju You should test my commit for this issue inside your VM instead of running the simulation on your Mac for a more accurate result

**Comment by poorva1209** [comment]
@shpoudel Shiva can you test the latest viz develop tag to see if your browser still shows you slowed down message. If so, we will re-open this issue.

**Comment by shpoudel** [comment]
@poorva1209 I was actually checking this while you were commenting on this issue here. And yes, I am still seeing the browser is slowing down with the same message.

**Comment by shpoudel** [comment]
@poorva1209 Just to make sure about pulling the latest develop tag. I did the following steps recently. 
./stop.sh -c 
docker-compose pull (Error shown below for this)
./run.sh -t develop

I am getting following errors while doing **docker-compose** pull command:

WARNING: The GRIDAPPSD_TAG variable is not set. Defaulting to a blank string.
Pulling redis (redis:3.2.11-alpine)...
ERROR: Get https://registry-1.docker.io/v2/: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)

So, I believe I am not running the latest develop tag, right? How can I get rid of that docker-compose pull command error?

**Comment by poorva1209** [comment]
@tonya1 I thought this was just something with my VM but Shiva is also getting this error above. Can you try stop.sh -c and then run.sh -t develop. For me the containers and specially the MySQL container is not removing at all.

**Comment by shpoudel** [comment]
I did /stop.sh -c and ./run.sh -t develop but the issue for browser is still there. 

Based on my experience, when pulling a latest tag, I usually get several pulling commands in the terminal if it is actually pulling the latest changes. But, now it just connects to the gridappsd container soon after doing run.sh -t develop. So, the latest develop tag is not pulled and browser issue still remains there in my local machine.

**Comment by tonya1** [comment]
@shpoudel I have tried to reproduce this on my vm without success.  Would you wan to have a Teams meeting and share your screen?

**Comment by poorva1209** [comment]
I tried to reproduce this error with latest develop tag and couldn't. I know I was getting this last week. @shpoudel What time would work for you for a quick a call today?

**Comment by shpoudel** [comment]
@poorva1209 @tonya1 What about this afternoon? Anytime after 1:00 PM.

**Comment by poorva1209** [comment]
@tonya1 Can you setup something with Shiva to debug this as I won't be working after 1pm?

**Comment by tonya1** [comment]
@poorva1209 @shpoudel I was able to replicate the browser slowdown when I changed my VM to 8GB of memory with 4CPU using Firefox to run the 9500 model for 1200 seconds.  Firefox stopped at 5 seconds with a prompt to wait. 

I was able to run chrome without the error but I encountered a lot of errors trying to start the platform with the limited resources.  I had to kill and start the ./run-gridappsd.sh multiple times to get things running correctly.

I tried going to 6 cpu or 10 GB of memory and still saw the firefox wait error.
